int32 iteration
float32 distance_total
float32 entropy_total
string method_generation
string method_selection
float32 time_iteration
float32 time_generation
float32 time_selection
float32 time_mapping
float32 time_termination
float32[] utilities
float32 utility_max
float32 utility_med
geometry_msgs/Pose selected_pose
